coalib/testing/BearTestHelper.py

```diff
@@ -0,0 +1,16 @@
+from unittest.case import skip, skipIf
+
+
+def generate_skip_decorator(bear):
+    """
+    Creates a skip decorator for a `unittest` module test from a bear.
+
+    `check_prerequisites` is used to determine a test skip.
+
```

Comment:
removed those comments since they were about the file and not the movement of those files , rebasing and pushing